controllers/storagecluster/generate.go

```diff
@@ -72,6 +72,10 @@ func generateNameForCephBlockPoolSC(initData *ocsv1.StorageCluster) string {
 	return fmt.Sprintf("%s-ceph-rbd", initData.Name)
 }
 
+func generateNameForVirtualCephBlockPoolSC(initData *ocsv1.StorageCluster) string {
+	return fmt.Sprintf("%s-ceph-rbd-virtualization", initData.Name)
```

Comment:
I prefer this name, but there is also `%s-ceph-non-resilient-rbd` below. Maybe call this `%s-ceph-virtualization-rbd` to match it a little more?

Reply:
Bike shedding a bit, but imagining a non-resilient virtualization storage class, a natural name for it would be `ceph-non-resilient-rbd-virtualization` (i.e. `virtualization` applies to `rbd`, not to `ceph`).  I like the original better.

Reply:
right, I guess a naming convention could be `${clustername}-ceph-${storage_type}-${workload}`

That makes `%s-ceph-non-resilient-rbd-kafka` and `%s-ceph-rbd-postgresql` useful and consistent names too.

:bike:

Reply:
Settling on %s-ceph-rbd-virtualization then.